wp-cli/vip-filesystem.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+use \WP_CLI;
+
+/**
+ * Helper command to manage VIP Go Filesystem
+ *
+ * @package Automattic\VIP\Files
+ */
+class VIP_Files_CLI extends \WPCOM_VIP_CLI_Command {
+
+	private $dry_run = false;
+
+	private $progress;
+
+	/**
+	 * Update file attachment metadata
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--dry-run=<dry-run>]
+	 * : Wether or not to update to database, or simply inspect it.
+	 * ---
+	 * default: false
+	 * options:
+	 *   - true
+	 *   - false
+	 * ---
+	 *
+	 * [--batch=<batch>]
+	 * : Batch size to process attachments in.
+	 * ---
+	 * default: 1000
+	 * ---
+	 *
+	 * ## EXAMPLES
+	 *     wp vip files update-filesizes
+	 *
+	 * @subcommand update-filesizes
+	 * @sypnosis [--dry-run=<dry-run>] [--batch=<batch>]
+	 */
+	public function update_filesizes( $args, $assoc_args ) {
+		global $wpdb;
+
+		$offset = 0;
+
+		WP_CLI::line( 'Updating attachment filesize metadata...' );
+
+		// Parse arguments
+		$_dry_run = WP_CLI\Utils\get_flag_value( $assoc_args, 'dry-run', true );
+		if ( 'false' === $_dry_run ) {
+			$this->dry_run = false;
+		}
```

Comment:
Should probably have a fallback here?

```suggestion
		} else {
			$this->dry_run = true;
		}
```

Reply:
Maybe I should just set the default value for `$dry_run` to `true` in line 12?

Reply:
Oh that's a better idea :+1: